rhino.yml

```diff
@@ -129,14 +153,17 @@ services:
 #    video-h264-encoder-amd64:
 #        container_name: video-h264-encoder-amd64
 #        build:
-#           context: https://github.com/chalmers-revere/opendlv-video-h264-encoder.git
+#            context: https://github.com/chalmers-revere/opendlv-video-h264-encoder.git
 #            dockerfile: Dockerfile.amd64
 #        restart: on-failure
 #        network_mode: "host"
 #        ipc: "host"
 #        volumes:
 #        - /tmp:/tmp
-#       command: "--cid=100 --name=video0.i420 --width=640 --height=480"
+#        cap_add:
+#        - SYS_NICE
+#        entrypoint: ""
+#        command: "--cid=100 --name=video0.i420 --width=640 --height=480"
 
     # Encoder for camera-opencv(vp8)
 #    video-vpx-encoder:

```

Comment:
The service `opendlv-video-vpx-encoder` must also match with `CLUON_SHAREDMEMORY_POSIX=?` !